### PR TITLE
[LM-384] Fix two bugs on lemonfrog.ch website

### PR DESCRIPTION
**Pull request by bobilo-qa**
This PR adds a bunch of device breakpoints, new logo, furthermore improve the text in de and en version (thanks to Charlotte for helping). I've also changed the structure of our css files to follow the advice of Mateusz. Now there are 3 files. The separated file 'normalize.css' with no changes and 2 new each per language with my declarations for mobiles. 

Don't know why there is now a new file .DS_Store because I have not added him. Must jump in during the commit.

### CODE REVIEW

**Comment by mateuszgorniak** [comment]
Please create `.gitignore` file and add `.DS_Store` as one of the excluded files - https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files

File: css/styles_en.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...tive}@media only screen and (max-width: 768px){.footer-main{padding:2.5rem 0 0 0}}.footer-main--page{padding:0}.footer-main__social-media{width:100%;margin:0;padding:7rem 0 7rem 0;background-color:#ececec;display:block}@media only screen and (max-width: 768px){.footer-main__social-media{padding:3.5rem 0 3.5rem 0}}.footer-main__social-media--home{background-color:#fff}.footer-main__social-media--home .social-media__icon:hover{background-color:#ececec}.footer-main__social-media-title{width:100%;max-width:564px;font-size:21px;font-weight:700;margin:0 auto 0 auto;padding:0 0 1.5rem 0;text-align:center;line-height:1.7;display:block;position:relative}@media only screen and (max-width: 992px){.footer-main__social-media-title{font-size:19px}}@media only screen and (max-width: 768px){.footer-main__social-media-title{font-size:17px}}@media only screen and (max-width: 576px){.footer-main__social-media-title{font-size:15px}}.footer-main__meta{width:100%;margin:0;padding:1.5rem 0 1.5rem 0;background-color:#191C1E;line-height:1;display:block}.footer-main__meta a{color:#fff;margin:0 .5rem 0 .5rem;text-decoration:none;display:inline-block}.footer-main__meta a:hover{color:#fff;text-decoration:none}
```
Comment (mateuszgorniak) [inline css/styles_en.css:1]: Do we have to copy the same content everywhere?

File: css/styles_en.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+@media only screen and (min-width: 800px)
```
Comment (mateuszgorniak) [inline css/styles_en.css:3]: You can work on indentations, use 2 spaces (1 tab) and group classes by the same parent node, like:
```
@media only screen and (min-width: 800px)
                   and (max-width: 1000px)
                   and (orientation: landscape) {
  .header-main__title {
     padding-top: 20px;
     margin-top: 46px;
  }
  .header-main__text {
    margin-bottom: 30px;
  }
}
```

File: css/styles_en.css (hunk 1 above)
Comment (mateuszgorniak) [reply css/styles_en.css:1]: As we discussed on Slack, please use separate `normalize.css` file and just add it to the main page `<link rel="stylesheet" href="css/normalize.css" type="text/css" media="all" />` to include the code.

**Review by mateuszgorniak — CHANGES_REQUESTED** [review]

**Review by bobilo-qa — COMMENTED** [review]
Didn't change the commit message but yeah hope now it's better 😉

File: css/styles.css [hunk 3]
```diff
@@ -1 +1,28 @@
...
+                   and (max-width: 1000px)
+                   and (orientation: landscape) {
+  .header-main__text{
+     margin-bottom: 30px;
+  }
+}
+
+@media only screen and (max-width: 320px)
+                   and (max-height: 568px)
+                   and (orientation: portrait)  {
```
Comment (mateuszgorniak) [inline css/styles.css:20]: Try to keep consistency with spacings - here you used one, `.header-main__text{` no spacing between class name and `{`, and below you use `.header-main__title {`

File: en/index.html [hunk 4]
```diff
@@ -34,7 +34,8 @@
 
     <link rel="canonical" href="https://www.lemonfrog.ch/en" />
 
-    <link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" />
+    <link rel="stylesheet" href="../css/normalize.css" type="text/css" media="all" />
```
Comment (mateuszgorniak) [inline en/index.html:37]: 👍

**Review by mateuszgorniak — CHANGES_REQUESTED** [review]
Please check spacing and remove `images/.DS_Store` from the repository.

**Review by mateuszgorniak — APPROVED** [review]

**Review by lemonfrog — APPROVED** [review]

**Pull request merged**